.travis.yml

```diff
@@ -13,32 +21,47 @@ python:
 # Manually define here the combinations environment variables to test
 # https://github.com/travis-ci/travis-ci/issues/1519
 env:
-  - TEST_MOBILE=True  DB=postgresql
-  - TEST_MOBILE=True  DB=sqlite
-  - TEST_MOBILE=False DB=postgresql
-  - TEST_MOBILE=False DB=sqlite
+  - TEST_MOBILE=True  DB=postgresql TRAVIS_NODE_VERSION="4"
+  - TEST_MOBILE=True  DB=sqlite     TRAVIS_NODE_VERSION="4"
+  - TEST_MOBILE=False DB=postgresql TRAVIS_NODE_VERSION="4"
+  - TEST_MOBILE=False DB=sqlite     TRAVIS_NODE_VERSION="4"
 
 # Install the application
 install:
-  - sh -c "if [ '$DB' = 'postgresql' ]; then pip install psycopg2; fi"
+  # Update nvm and set wanted Node version.
+  # We update nvm using the script method instead of git, which is selected
+  # automatically, as git won't work because the $HOME/.nvm is not a git
+  # repository and the directory is not empty.
+  - curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.4/install.sh | METHOD=script bash
+  - . $HOME/.nvm/nvm.sh
+  - nvm install $TRAVIS_NODE_VERSION
+  - nvm use $TRAVIS_NODE_VERSION
```

Comment:
You can't tell Travis to use a different version unfortunately.

The nvm version installed on Travis is very old, 0.23.3 released on Feb 2, 2015. This part updates nvm to version 0.31.4.